src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
@bjwatson We wouldn't need the `gccl` key exactly, but we would still like to know what version of `google-cloud` is in use that contains the GAPIC client. The reason is that with that information, we can easily compare how up to date our users are. Having only the generator version doesn't allow a good apples to apples comparison of how up to date they are. Having the same version under `gccl` and `gapic` is redundant (which is what drove my first reaction), but the presence or absence of those keys still conveys information (whether the handwritten layer is used, and whether the GAPIC client is used).